x-pack/platform/plugins/shared/osquery/server/plugin.ts

```diff
@@ -168,9 +181,17 @@ export class OsqueryPlugin implements Plugin<OsqueryPluginSetup, OsqueryPluginSt
                   }));
 
                 if (allPacks.saved_objects) {
```

Comment:
@tomsonpl 
Could you take a look and let me know if `allPacks.saved_objects.length` wouldn't make more sense here?

Reply:
Sure 👍